roles/openstack_health_monitor/defaults/main.yml

```diff
@@ -55,3 +55,5 @@ openstack_health_monitor_image: "{{ docker_registry_openstack_health_monitor }}/
 openstack_health_monitor_container_name: openstack_health_monitor
 
 openstack_health_monitor_network: 172.31.100.160/28
+
+openstack_health_monitor_service_name: "docker-compose@openstack_health_monitor"
```

Comment:
Not sure whether an extra variable for this is useful, the@xxx part at least always needs to match the docker_compose_directory above.

Reply:
We use a dedicated variable everywhere else for this. We can cleanup this in a separate PR.

Reply:
https://github.com/osism/issues/issues/375